uwblueprint/childrens-aid-society
# Set Up Design System Spacing

## Notion ticket link

[Set Up Design System Spacing](https://www.notion.so/uwblueprintexecs/Set-up-design-system-spacing-bc9ba5ed86e24149a65d32ee05816d1b)



## Implementation description
* Implemented spacing for margin/padding within the theme/ folder
* Now can easily implement spacing wherever needed



## Steps to test
1. run localhost:3000
2. see on home page the new spacing at the bottom (3 new rectangles with the respective spacing)



## What should reviewers focus on?
* See if the spacing works, do the rectangles pop up with the appropriate spacing


## Checklist
